### PR TITLE
[Relay]Release preview version.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/EventGrid/EventGrid.Test/ScenarioTests/Common.ps1 [hunk 1]
```diff
@@ -169,7 +169,7 @@ function New-HybridConnection($ResourceGroupName, $NamespaceName, $HybridConnect
     Write-Debug "Creating namespace $NamespaceName in resource group $ResourceGroupName and location $Location"
     New-AzRelayNamespace -ResourceGroupName $ResourceGroupName -Name $NamespaceName -Location $Location
     Write-Debug "Creating hybridconnection $HybridConnectionName in Namespace $NamespaceName in resource group $ResourceGroupName and location $Location"
-    New-AzRelayHybridConnection -ResourceGroupName $ResourceGroupName -Namespace $NamespaceName -Name $HybridConnectionName -RequiresClientAuthorization $True
+    New-AzRelayHybridConnection -ResourceGroupName $ResourceGroupName -Namespace $NamespaceName -Name $HybridConnectionName -RequiresClientAuthorization
```
Comment (wyunchi-ms) [inline src/EventGrid/EventGrid.Test/ScenarioTests/Common.ps1:172]: Is `RequiresClientAuthorization` still work for `New-AzRelayHybridConnection`?

Comment (LucasYao93) [reply src/EventGrid/EventGrid.Test/ScenarioTests/Common.ps1:172]: Yes, It's still work.

File: tools/Az/Az.psd1 [hunk 2]
```diff
@@ -111,7 +111,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.11.0'; },
                @{ModuleName = 'Az.RecoveryServices'; RequiredVersion = '6.1.2'; }, 
                @{ModuleName = 'Az.RedisCache'; RequiredVersion = '1.7.0'; }, 
                @{ModuleName = 'Az.RedisEnterpriseCache'; RequiredVersion = '1.1.0'; }, 
-               @{ModuleName = 'Az.Relay'; RequiredVersion = '1.0.3'; }, 
+               #@{ModuleName = 'Az.Relay'; RequiredVersion = '2.0.0'; },
```
Comment (BethanyZhou) [inline tools/Az/Az.psd1:114]: Update AzPreview.ps1 as well.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

**Comment by BethanyZhou** [comment]
Close as https://github.com/Azure/azure-powershell/pull/20993/files opend

**Pull request closed without merging**